packages/grpc-js/src/transport.ts

```diff
@@ -101,28 +101,29 @@ class Http2Transport implements Transport {
   /**
    * The amount of time in between sending pings
    */
-  private keepaliveTimeMs = -1;
+  private readonly keepaliveTimeMs: number;
   /**
    * The amount of time to wait for an acknowledgement after sending a ping
    */
-  private keepaliveTimeoutMs: number = KEEPALIVE_TIMEOUT_MS;
+  private readonly keepaliveTimeoutMs: number;
   /**
-   * Timer reference for timeout that indicates when to send the next ping
+   * Indicates whether keepalive pings should be sent without any active calls
+   */
+  private readonly keepaliveWithoutCalls: boolean;
+  /**
+   * Timer reference indicating when to send the next ping or when the most recent ping will be considered lost.
    */
-  private keepaliveTimerId: NodeJS.Timeout | null = null;
+  private keepaliveTimeout: NodeJS.Timeout | null = null;
```

Comment:
This shouldn't be called "keepaliveTimeout" because that is already the name of another concept in this context. `keepaliveTimer` should be fine.